src/Cooked/InitialDistribution.hs

```diff
@@ -1,12 +1,15 @@
 -- | This module provides a convenient way to spread assets between wallets and
 -- scripts at the initialization of the mock chain. These initial assets can be
--- accompanied by datums and reference scripts.
+-- accompanied by datums, staking credentials and reference scripts.
 module Cooked.InitialDistribution
   ( InitialDistribution (..),
     distributionFromList,
+    toInitDistWithMinAda,
   )
```

Comment:
I think it might come in handy (in which case we should document the function in the module)

```suggestion
module Cooked.InitialDistribution
  ( InitialDistribution (..),
    distributionFromList,
    toInitDistWithMinAda,
    toPaysList,
  )
```